elasticsearch/init/0520-bz1658632-remove-old-sg-indices

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# Copyright 2018 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# init script to cleanup old searchguard indices
+# https://bugzilla.redhat.com/show_bug.cgi?id=1658632
+set -e
+
+if [ -n "${DEBUG:-}" ] ; then
+  set -x
+fi
+
+source "logging"
+
+script=$(basename $0)
+
+info "Starting init script: ${script}"
+
+routing_allocation(){
```

Comment:
nothing calls this?

Reply:
correcting. got lost in the merge someplace